tests/functional/spec/supplementary_data_employees/supplementary_data_employees.spec.js

```diff
@@ -0,0 +1,179 @@
+import { click, assertSummaryTitles } from "../../helpers";
```

Comment:
The name of the spec is probably incorrect, employees are not important here, it's which functionality we test which is `supplementary data with list collector`, how they work together. Or `supplementary data with repeating section` but the repeating section is driven by/caused by list collector.